third_party/move/tools/move-asm/tests/assembler/branch.exp

```diff
@@ -3,24 +3,21 @@ processed 1 task
 task 0 'publish'. lines 1-14:
 
 == BEGIN Bytecode ==
-// Move bytecode v7
-module 66.test {
+// Bytecode version v7
+module 0x66::test
+// Function definition at index 0
+fun f(l0: u64)
+l1: copy_loc l0
+    ld_u64 0
+    gt
+    br_false l0
```

Comment:
nit: seems lake labels and locals are all called as `l*`? Maybe use `bb` for labels as if basic block?